### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Edit-Web Application_Screen Reader: Narrator/NVDA is not reading the label of the input elements of the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#164]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated**: MAS1.3.1

Repro Steps:
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" commands button of the existing website inside the grid and Press Enter.
6. Tab till "Edit" in the expanded menu and Press Enter.
7. Tab till "Web Applications" in the left menu section and Press Enter.
8. Tab till "Create Web Application" button and Press Enter.
9. Navigate to Application Pool page tab.
10. Verify that screen reader is reading the label of all the input form elements of the page.

**Actual Result:**
Narrator/NVDA is not reading the label of the following input form elements:
"Use Custom Application Pool" & "Auto Start" toggle button, "Identity" dropdown, "Name" action button, "Pipeline" and ".Net Framework" list item.

**Expected Result:**
Screen Reader should read the label of all the input form elements of the page.

**Note:** Similar issue is observed in "Virtual Directories" page as well, where Narrator/NVDA is not reading the label of any of the form fields.

**User Impact:**
Screen reader users will be unable to know the labels of the form field and hence they will not be able to provide inputs to the form fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug 164_screen reader is not reading the label of the input elements (Virtual Directories).pptx](https://github.com/Microsoft/IIS.WebManager/files/2332100/Bug.164_screen.reader.is.not.reading.the.label.of.the.input.elements.Virtual.Directories.pptx)
[Bug 164_screen reader is not reading the label of the input elements (web applications Page).pptx](https://github.com/Microsoft/IIS.WebManager/files/2332101/Bug.164_screen.reader.is.not.reading.the.label.of.the.input.elements.web.applications.Page.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
